app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
```

Comment:
probably, customers shouldn't be allowed to edit anyone else, except themselves, and destroy someone at all: 
imagine, a customer has deleted admin. sound ridiculous

`pandit` and policies have to help

Reply:
also, don't miss the part about configurable parameters by policies: `role` should be excluded from the parameters list in case of customer user